charts/camunda-platform-alpha/templates/zeebe-gateway/deployment.yaml

```diff
@@ -131,7 +131,11 @@ spec:
           {{- if .Values.zeebeGateway.readinessProbe.enabled }}
           readinessProbe:
             httpGet:
-              path: {{ .Values.zeebeGateway.contextPath }}{{ .Values.zeebeGateway.readinessProbe.probePath }}
+              path: {{ if eq .Values.zeebeGateway.contextPath "/" }}
```

Comment:
Let's move that to a named template in `_helpers.tpl`.
Also please add a unit test for separated Ingress since we only test the combined Ingress in the integration test.

Reply:
defined a new helper template `camundaPlatform.zeebeGatewayReadinessProbePath` and created the unit test will update the ticket with the changes.